### PR TITLE
Feature add circle ci and GitHub actions

### PR DESCRIPTION
**Pull request by OleksaRiabukha**
Each commit now triggers Circle CI rspec tests and pull requests can't be merged automatically if CI fails or there are no reviews.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    docker:
+      - image: circleci/ruby:3.0.1-node
+        environment:
+          DATABASE_URL: postgresql://postgres:secret@localhost:5432
+          POSTGRES_HOST: localhost
+          POSTGRES_USER: postgres
+          POSTGRES_DB: the_best_app_test
+          POSTGRES_PASSWORD: password
+      - image: circleci/postgres:12.7
+        environment:
```
Comment (BogdanBusko) [inline .circleci/config.yml:52]: Do we really need this?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
+version: 2.1
+jobs:
+  setup:
+    docker:
+      - image: circleci/ruby:3.0.1-node
+        environment:
+          NODE_ENV: test
+          RAILS_ENV: test
+          POSTGRES_HOST: localhost
```
Comment (BogdanBusko) [inline .circleci/config.yml:9]: Do we need this Postgres config here

Comment (OleksaRiabukha) [reply .circleci/config.yml:9]: Yes, without it   "run: bundle exec rails webpacker:compile" fails with key_error. Was able to resolve it after adding config there

File: .circleci/config.yml (hunk 1 above)
Comment (OleksaRiabukha) [reply .circleci/config.yml:52]: Yes, we found out that those env are need during the call. And env after are also needed as postgres image didn't work without password and user

**Review by BohdanBusko — APPROVED** [review]

**Pull request merged**